pennylane_lightning/core/src/simulators/lightning_kokkos/gates/GateFunctorsParam.hpp

```diff
@@ -55,41 +60,46 @@ template <class Precision> struct multiQubitOpFunctor {
             wires_host(wires_.data(), wires_.size());
         Kokkos::resize(wires, wires_host.size());
         Kokkos::deep_copy(wires, wires_host);
-        dim = static_cast<std::size_t>(1U) << wires_.size();
+        dim = one << wires_.size();
         num_qubits = num_qubits_;
         arr = arr_;
         matrix = matrix_;
+        std::pair<KokkosIntVector, KokkosIntVector> tmp =
+            wires2Parity(num_qubits_, wires_);
+        parity = tmp.first;
+        rev_wire_shifts = tmp.second;
```

Comment:
You may be able to do:
```suggestion
        std::tie(parity, rev_wire_shifts) = wires2Parity(num_qubits_, wires_);
```

Reply:
Neat, thanks!